pkg/vpcmodel/nodesExplainability.go

```diff
@@ -177,7 +184,8 @@ func (c *VPCConfig) computeExplainRules(srcNodes, dstNodes []Node,
 			if err != nil {
 				return nil, err
 			}
-			rulesThisSrcDst := &srcDstDetails{src, dst, common.NewConnectionSet(false), nil, nil, rulesOfConnection, nil}
+			rulesThisSrcDst := &srcDstDetails{src, dst, false, false, false,
```

Comment:
most values are default:
```suggestion
			rulesThisSrcDst := &srcDstDetails{src:src, dst:dst ...
```

Reply:
Right; how do you suggest to improve the code? 
(the suggested code is illegal)